File: src/main/java/com/github/eendroroy/sdk/openweather/interceptor/DefaultOWInterceptor.java
package com.github.eendroroy.sdk.openweather.interceptor;

import okhttp3.Request;
import okhttp3.Response;

/**
 * @author indrajit
 */

public class DefaultOWInterceptor extends AbstractOWInterceptor {
    @Override
    public Request interceptRequest(Request request) {
        return request;
    }

    @Override
    public Response interceptResponse(Response response) {
        return response;
    }
}

File: src/main/kotlin/com/github/eendroroy/sdk/openweathermap/interceptor/DefaultOWInterceptor.kt
package com.github.eendroroy.sdk.openweathermap.interceptor

import okhttp3.Request
import okhttp3.Response

/**
 * @author indrajit
 */
class DefaultOWInterceptor : AbstractOWInterceptor() {
    override fun interceptRequest(request: Request): Request {
        return request
    }

    override fun interceptResponse(response: Response): Response {
        return response
    }
}